### PR TITLE
bump version for v6.5.2

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -554,11 +554,11 @@ tiup cluster audit
 Starting component `cluster`: /home/tidb/.tiup/components/cluster/v1.11.3/cluster audit
 ID      Time                       Command
 --      ----                       -------
-4BLhr0  2023-03-10T13:25:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.1 /tmp/topology.yaml
-4BKWjF  2023-03-10T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.1 /tmp/topology.yaml
-4BKVwH  2023-03-10T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.1 /tmp/topology.yaml
+4BLhr0  2023-03-10T13:25:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v6.5.2 /tmp/topology.yaml
```
Comment (qiancai) [inline tiup/tiup-cluster.md:557]: 日期要更新为 6.5.2 发布的日期吗

**Review by qiancai — APPROVED** [review]
other LGTM

File: tiup/tiup-cluster.md (hunk 1 above)
Comment (ran-huang) [reply tiup/tiup-cluster.md]: done

**Pull request merged**